toolkit/imageconfigs/core-efi.json

```diff
@@ -49,10 +49,15 @@
                 "packagelists/core-packages-image.json",
                 "packagelists/cloud-init-packages.json"
             ],
+            "Packages": [
+                "wget"
+            ],
```

Comment:
Is this intended or just a test leftover?

Reply:
ah, good catch. Was just a leftover